fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyServerPropertiesMgr.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.derby;
+
+import java.util.Properties;
+
+
+/**
+ * Server properties for embedded derby which is used in unit tests and server integration tests,
+ * equal to setting in derby.properties.
+ */
+public class DerbyServerPropertiesMgr {
+    public static void setServerProperties (boolean isDebug) {
+        Properties sysProperties = System.getProperties();
```

Comment:
does Derby automatically write this out when it's set on System.getProperties?

Reply:
no, it doesn't.

Reply:
How does this write out? ... 

`try (PrintWriter out = new PrintWriter(DERBY_PROPERTIES)){ `

Is very different from System.getProperties

Reply:
yes, you are correct. I had used the perperties file because I did find this way at that time, now I found this way is much easier and don't have to associate with any directory.

Reply:
Excellent.  thanks